rules/choosing-authentication/rule.md

```diff
@@ -140,28 +127,58 @@ Advantages:
 * Allows you to define custom authentication logic
```

Comment:
Same comment as above.  Can you provide some more details around what 'custom logic means'?

Reply:
Agreed - done.